lib/Dialect/TTIR/Transforms/Passes.cpp

```diff
@@ -284,27 +285,39 @@ class TTIRGenericRegionRewriter
 
     auto dps = cast<DestinationStyleOpInterface>(op.getOperation());
 
-    // Create a dispatch op
+    // Create a generic op.
     auto [indexingMaps, iteratorTypes] = op.getIndexingMaps(rewriter);
     auto constraints = rewriter.getArrayAttr(SmallVector<Attribute>(
         op->getNumOperands(), rewriter.getAttr<OperandConstraintAttr>(
                                   OperandConstraint::AnyDeviceTile)));
-    auto dispatch = rewriter.create<ttir::GenericOp>(
+
+    // For testing purposes try getting grid of the resulting tensor and put the
```

Comment:
@nsmithtt  do we have such flag? 
I couldn't force generic op's grid through mlir test file.

Reply:
I was trying to work on a simple analysis pass for us that always picks largest grid size / supports overrides.  I'll try and make some progress this weekend.

Reply:
@nsmithtt Do we need to sync on this? LMK

Reply:
Yeah we can! We need something that just picks max grid for all ops, but it needs to work for ttir.generic, it's kind of different than the existing analysis passes.